### PR TITLE
Using input field for monitoring endpoint url

### PR DESCRIPTION
**Pull request by patschi**
Simply replaced the monitoring endpoint url with an input field and added a listener for automatically selecting the whole url when clicking on the input field.

image

Should (partially) solve #90 point 4, clipboard copy functionality may still be missing.

### CODE REVIEW

**Comment by Noodlesalat** [comment]
Hey @patschi ,
do you still want to integrate the clipboard copy functionality? I found a example in the Nextcloud Server code but it did not worked out oft the box for me. [Link](https://github.com/nextcloud/server/blob/23785d108d6f1a5641bad2cf91ae3c696aac0ef0/settings/templates/settings/personal/security.php#L101)
If you do not habe the time to add this I would try to do so by forking your repo and working on a pull request.

**Comment by patschi** [comment]
Thanks a lot for pointing that out, @Noodlesalat! I have just pushed some new changes, including the clipboard copy feature.

Now it looks like:
image

File: templates/settings-admin.php [hunk 1]
```diff
@@ -70,5 +71,10 @@
 <div class="section" id="ocsEndPoint">
 	<h2><?php p($l->t('External monitoring tool'));?></h2>
 	<p>
-		<?php p($l->t('You can connect an external monitoring tool by using this end point: ') . $_['ocs']);?>
+		<?php p($l->t('You can connect an external monitoring tool by using this end point:')); ?>
+	</p>
+	<div>
+		<input type="text" readonly="readonly" style="width: 415px;" id="monitoring-endpoint-url" value="<?php echo p($_['ocs']); ?>" />
```
[the remark this answers is not in the document]
Comment (patschi) [reply templates/settings-admin.php]: Very good points, thanks! 80% works out great on a few tests, I think it's a good value IMO.

**Comment by Noodlesalat** [comment]
This looks good for me now. In my opinion this can be merged! @MorrisJobke
Thanks @patschi for your work!

**Pull request merged**